src/RemoteAPI/content-suggestions/class-content-suggestions-base-api.php

```diff
@@ -142,4 +162,31 @@ protected function post_request( array $query = array(), array $body = array() )
 
 		return $decoded;
 	}
+
+	/**
+	 * Truncate the content of an array to a maximum length.
+	 *
+	 * @param string|array|mixed $content The array to truncate.
+	 * @return string|array|mixed The truncated array.
+	 */
+	public function truncate_array_content( &$content ) {
```

Comment:
What's the thought behind passing this by reference here? Asking because I also see return statements in the function.

Reply:
Yeah that's a fair question, I thought I would need to access it by reference initially, and forgot to remove the `&` afterwards. Removed :)